tests/seechange_config_test.yaml

```diff
@@ -20,6 +20,9 @@ archive:
   local_write_dir: null
   token: insecure
 
+catalog_gaiadr3:
+  fallback_datalab: False
```

Comment:
Why not set this to True by default? Could be useful to have our tests not fail when the server is down...

Reply:
Yeah, will do.  (Obviously, had this there in the first place to make sure I was testing the new server.)